public/userInfo.php

```diff
@@ -285,8 +285,8 @@ function resize() {
             $totalTruePoints = $userMassData['TotalTruePoints'];
 
             $retRatio = sprintf("%01.2f", $totalTruePoints / $totalHardcorePoints);
-            echo "Hardcore Points: $totalHardcorePoints <span class='TrueRatio'> ($totalTruePoints)</span></span><br>";
-            echo "Hardcore Achievements: $totalHardcoreAchievements<br>";
+            echo "Hardcore Points: " . localized_number($totalHardcorePoints) . "<span class='TrueRatio'> (" . localized_number($totalTruePoints) . ")</span></span><br>";
```

Comment:
This is unrelated to the PR, but I am just noticing that there's a bug on this line of code we should probably clean up while we're in here.

It appears there's an extra closing `</span>` tag here, possibly for something that used to exist but was removed.

Reply:
Extra `span` removed